orchestration/dagster_orchestration/hca_manage/diff_dirs.py

```diff
@@ -12,7 +12,8 @@
 logging.basicConfig(level=logging.INFO)
 
 
-def run(project: str, source_bucket: str, source_prefix: str, target_bucket: str, target_prefix: str) -> None:
+def diff_dirs(project: str, source_bucket: str, source_prefix: str,
+              target_bucket: str, target_prefix: str) -> tuple[dict[str, int], dict[str, int]]:
```

Comment:
my preference for function indentation is either "all on one line" or "one line per argument" with no in between, for the sake of being able to easily identify where arguments start and stop.